src/EditorFeatures/Test/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -260,21 +290,34 @@ protected virtual bool CompareItems(string actualItem, string expectedItem)
             string expectedItemOrNull, string expectedDescriptionOrNull,
             SourceCodeKind sourceCodeKind,
             bool usePreviousCharAsTrigger, bool checkForAbsence,
-            int? glyph, int? matchPriority)
+            int? glyph, int? matchPriority, IEnumerable<KeyValuePair<OptionKey, object>> options)
         {
             Glyph? expectedGlyph = null;
             if (glyph.HasValue)
             {
                 expectedGlyph = (Glyph)glyph.Value;
             }
 
-            var document1 = await WorkspaceFixture.UpdateDocumentAsync(code, sourceCodeKind);
-            await CheckResultsAsync(document1, position, expectedItemOrNull, expectedDescriptionOrNull, usePreviousCharAsTrigger, checkForAbsence, expectedGlyph, matchPriority);
+            bool dispose;
+            var workspace = await GetWorkspaceAsync(out dispose, options: options);
```

Comment:
Is this just so we restore teh workspace back to its original state?  if so, we should capture the options before, then set the options, then restore them.